processing/src/main/java/io/druid/query/groupby/GroupByQuery.java

```diff
@@ -120,12 +120,15 @@ public GroupByQuery(
       Preconditions.checkArgument(spec != null, "dimensions has null DimensionSpec");
     }
     this.aggregatorSpecs = aggregatorSpecs == null ? ImmutableList.<AggregatorFactory>of() : aggregatorSpecs;
-    this.postAggregatorSpecs = postAggregatorSpecs == null ? ImmutableList.<PostAggregator>of() : postAggregatorSpecs;
+    this.postAggregatorSpecs = Queries.prepareAggregations(
```

Comment:
Seems like `postAggregatorSpecs == null ? ImmutableList.<PostAggregator>of() : postAggregatorSpecs` is a common pattern, might as well have `Queries.prepareAggregations` do this.